forem/forem
# Adds a new admin setting for experience levels.

## What type of PR is this? (check all applicable)


## Description
These admin settings are used in the post management UI to determine the range of experience. These values fallback to the ones we currently see in DEV, "Total Newbies" and "Senior Devs". There is also a removal of the word coding when talking about experience level, since the context for each forem instance should be implied when referencing experience.

## Related Tickets & Documents
Closes #10845 

## QA Instructions, Screenshots, Recordings

- Verify that the fallback values of 'Total Newbies' and 'Senior Devs' are present before testing the new functionality
- As an admin with a config update role navigate to /admin/config
- Toggle the Community Content section and observe there are two new fields at the bottom, Experience Low/High
- Update those values with any text and save the config
- Then, create a post and publish it
- In your personal dashboard, click manage to view a post's experience level and observe that the saved config values are there. There should be no changes in setting the experience level of a post.
- Navigate to /settings/ux and observe that under Content Customization the text now reads `What is your approximate experience level (1-5)?`

UX Settings:

![Screen Shot 2020-10-15 at 3 26 01 PM](https://user-images.githubusercontent.com/12255511/96176851-edd98300-0efa-11eb-8830-a9fb0823ea63.png)

Admin Config:

![Screen Shot 2020-10-15 at 3 26 32 PM](https://user-images.githubusercontent.com/12255511/96176859-f16d0a00-0efa-11eb-949b-4c8a6b3f3417.png)

Post Management:

![Screen Shot 2020-10-15 at 3 27 00 PM](https://user-images.githubusercontent.com/12255511/96176870-f631be00-0efa-11eb-9b12-956cc3ac9bfd.png)


## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

_Edit: We need to add the values we want to display in DEV's admin dashboard to keep "Senior Devs" instead of the newly included defaults._

## [optional] What gif best describes this PR or how it makes you feel?

![kim kardashian west saying "I wish everyone could experience this"](https://media.giphy.com/media/kHT8KGUq30MRbl5gAJ/giphy.gif)
